css/style.css

```diff
@@ -46,14 +46,48 @@ body {
 }
 
 .menu__item {
-  padding: 0 13px;
-  font-family: 'Roboto', sans-serif;
-  font-weight: 500;
-  font-size: 18px;
+  position: relative;
+  margin-right: 26px;
+  font: 500 18px/21px 'roboto-medium', sans-serif;
+  color: #000;
+  text-decoration: none;
+  transition: opacity .5s;
+}
+
+.menu__item_active::after {
+  position: absolute;
+  display: block;
+  content: '';
+  left: 0;
+  width: 100%;
+  height: 3px;
+  background: #0056ec;
+}
+
+.menu__item::after {
+  position: absolute;
+  display: block;
+  content: '';
+  left: 0;
+  width: 0;
+  height: 3px;
+  background: #0056ec;
+  opacity: 0;
+  transition: opacity .5s, width .5s;
+}
+
+.menu__item:hover {
+  opacity: .5;
+}
+
+.menu__item:hover::after, 
```

Comment:
Codacy found an issue: [Unexpected whitespace at end of line (no-eol-whitespace)](https://app.codacy.com/gh/WebHeroSchool/html_abrmhah/pullRequest?prid=6313902)